app/controllers/application_controller.rb

```diff
@@ -75,4 +75,25 @@ def set_no_cache_header
   def touch_current_user
     current_user.touch
   end
+
+  def log_image_data_to_datadog
+    images = Array.wrap(params.dig("user", "profile_image") || params["image"])
+
+    raise if images.empty?
```

Comment:
Might be a good idea to raise the original error that we captured here instead of a generic RuntimeError.

Reply:
On both raises `Errno::ENAMETOOLONG` (the original error) is still raised by calling just `raise`. Should I explicitly raise `Errno::ENAMETOOLONG` instead?

Reply:
Oh neat! I should have guessed that given the tests, I assumed a RuntimeError would be raised since raise has no arguments.

Reply:
A bare `raise` inside a `rescue` will reraise the original exception, the same is true in a `rescue_from`:

```ruby
begin
  raise 1 / 0
rescue
  puts "oops"
  raise
end

oops
ZeroDivisionError (divided by 0)
```